packages/network/src/streamr-client-protocol.ts

```diff
@@ -0,0 +1,3 @@
+import Protocol from 'streamr-client-protocol'
+export * from 'streamr-client-protocol'
+export default Protocol
```

Comment:
I feel like there is a need to get rid of the instanceof checking for messages in the protocol to avoid having to strictly use the network's protocol instance in the broker and elsewhere.